test/test_rpc.py

```diff
@@ -348,6 +380,49 @@ def test_py_no_return_result(self):
         ret = dist.rpc("worker{}".format(dst_rank), no_result)
         self.assertEqual(ret, no_result())
 
+    @_wrap_with_rpc
+    def test_py_tensors(self):
+        n = self.rank + 1
+        dst_rank = n % self.world_size
+        ret = dist.rpc("worker{}".format(dst_rank),
+                       my_tensor_function,
+                       args=(torch.ones(n, n), torch.ones(n, n)))
+        self.assertEqual(ret,
+                         my_tensor_function(torch.ones(n, n),
+                                            torch.ones(n, n)))
+
+    @_wrap_with_rpc
+    def test_py_tensors_multi_async_call(self):
+        futs = []
+        n = self.rank + 1
+        dst_rank = n % self.world_size
+        for i in range(10):
```

Comment:
any reason to make this range(10) instead of say range(100)? just making it more likely to exercise race conditions.